Setup/UpgradeSchema.php

```diff
@@ -59,8 +59,8 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
             $this->updateSchemaVersion204($setup);
         }
 
-        if (version_compare($context->getVersion(), '2.0.7', '<')) {
-            $this->updateSchemaVersion207($setup);
+        if (version_compare($context->getVersion(), '2.1.0', '<')) {
+            $this->updateSchemaVersion210($setup);
```

Comment:
No let's keep 2.1.0 but 2.0.7 will be never released the next release will be 2.2.0 or 2.1.1.